### PR TITLE
Update links to reflect move from danielquinn to the-paperless-project

### PR DESCRIPTION
**Pull request by JensPfeifle**
I noticed all of the links in the readme, docs, etc are still referring to `danielquinn/paperless`. This is a suggestion to update them to `the-paperless-project/paperless` were applicable.

Unchanged links:
- The link for the coverage status badge (https://coveralls.io/repos/github/danielquinn/paperless/badge.svg?branch=master) cannot be updated without breaking the badge.
- The link behind the gitter badge (https://gitter.im/danielquinn/paperless)
- The link to Dockerhub (https://hub.docker.com/r/danielquinn/paperless/)

### CODE REVIEW

**Comment by stgarf** [comment]
@JensPfeifle Looks like travis failed [(because of itself)](https://travis-ci.org/the-paperless-project/paperless/jobs/502255845). Can you close/re-open the PR (just two one-second button clicks -- no need to copy/paste or re-file or anything)?

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+[![Documentation](https://readthedocs.org/projects/paperless/badge/?version=latest)](https://paperless.readthedocs.org/) [![Chat](https://badges.gitter.im/the-paperless-project/paperless.svg)](https://gitter.im/danielquinn/paperless) [![Travis](https://travis-ci.org/the-paperless-project/paperless.svg?branch=master)](https://travis-ci.org/the-paperless-project/paperless) [![Coverage Status](https://coveralls.io/repos/github/danielquinn/paperless/badge.svg?branch=master)](https://coveralls.io/github/danielquinn/paperless?branch=master) [![Thanks](https://img.shields.io/badge/THANKS-md-ff69b4.svg)](https://github.com/the-paperless-project/paperless/blob/master/THANKS.md)
```
Comment (ahyear) [inline README.md]: there are still references to danielquinn. is it normal?
@danielquinn can you also change de coveralls configuration tu reflect the good url?

```suggestion
[![Documentation](https://readthedocs.org/projects/paperless/badge/?version=latest)](https://paperless.readthedocs.org/) [![Chat](https://badges.gitter.im/the-paperless-project/paperless.svg)](https://gitter.im/danielquinn/paperless) [![Travis](https://travis-ci.org/the-paperless-project/paperless.svg?branch=master)](https://travis-ci.org/the-paperless-project/paperless) [![Coverage Status](https://coveralls.io/repos/github/the-paperless-project/paperless/badge.svg?branch=master)](https://coveralls.io/github/the-paperless-project/paperless?branch=master) [![Thanks](https://img.shields.io/badge/THANKS-md-ff69b4.svg)](https://github.com/the-paperless-project/paperless/blob/master/THANKS.md)
```

Comment (JensPfeifle) [reply README.md]: Links referencing the paperless project don't exist yet for the services mentioned in the original comment (or I wasn't able to find them). That is: coveralls, gitter, and dockerhub.
Once the configurations are updated/moved these can be updated as well.

Comment (ahyear) [reply README.md]: I changed the coveralls token to reflect the right repo.
I just have to wait the next build to make sure it's ok.

Comment (ahyear) [reply README.md]: Nevermind i'll propose the change myself...

**Comment by JensPfeifle** [comment]
Sorry @ahyear , didn’t immediately see your comment. 
I’ll update the coveralls link later today.

**Comment by ahyear** [comment]
no worries ;)

**Comment by JensPfeifle** [comment]
Ok. Remaining references are:

- The link to gitter (https://gitter.im/danielquinn/paperless)
- The link to Dockerhub (https://hub.docker.com/r/danielquinn/paperless/)

In my opinion that's okay. No real advantage to moving the Gitter chat, and the docker image location is not changing as far as I know (?).

**Comment by stgarf** [comment]
> Ok. Remaining references are:
> 
> * The link to gitter (https://gitter.im/danielquinn/paperless)
> * The link to Dockerhub (https://hub.docker.com/r/danielquinn/paperless/)
> 
> In my opinion that's okay. No real advantage to moving the Gitter chat, and the docker image location is not changing as far as I know (?).

@JensPfeifle I think the docker build should move to a new org in dockerhub, e.g. the-paperless-project.

There was work somewhere (can't find the issue at the moment) that was attempting to update the image on DockerHub (which is *woefully* out of date). This really should be fixed...

**Pull request merged**